### ISSUE TITLE
Design new run on Moderne button

### ISSUE THREAD

**Issue by mike-solomon** [openrewrite/rewrite-docs#317]
Right now, we have a button on each recipe that says: `Run this recipe with the Moderne SaaS`. [Example link](https://docs.openrewrite.org/recipes/ai/findcommentslanguage#see-how-this-recipe-works-across-multiple-open-source-repositories)

![image](https://github.com/user-attachments/assets/2c3c8f61-be2b-48a4-a770-c93658035714)

Olga suggested we rephrase this to: `Run this recipe on OSS repos at scale with the Moderne SaaS`.

I'd like a similar style image - just with this new phrasing.

**Comment by mike-solomon** [comment]
Thanks for the suggestion @LindsayAuch 

I think that looks really good -- but it would be impractical to have a unique photo per recipe. Do you have any ideas of what else we might do? Maybe there's a generic photo we could use instead?

**Comment by LindsayAuch** [comment]
How about using the 'Activity view for OpenSource'?

<img width="1329" alt="Screenshot 2024-11-07 at 12 50 36 PM" src="https://github.com/user-attachments/assets/6095dbb9-93c3-43d9-9697-0a6adb92cde8">

<img width="813" alt="Screenshot 2024-11-07 at 12 51 42 PM" src="https://github.com/user-attachments/assets/0038fbe3-b6df-405b-a295-0919bb2d6b9b">

**Comment by LindsayAuch** [comment]
[buttons.zip](https://github.com/user-attachments/files/17682134/buttons.zip)

Unselected state: ![runrecipeherebutton](https://github.com/user-attachments/assets/3e8ff877-c5f1-4d7d-8a2f-d371f6b4a662)
Selected state: ![runrecipeherebuttonselected](https://github.com/user-attachments/assets/a436a735-b82c-43ea-a362-ed9fd5225d06)

**Comment by mike-solomon** [comment]
Marking this as completed. Here's an example page where you can see this: https://docs.openrewrite.org/running-recipes/popular-recipe-guides/common-static-analysis-issue-remediation

The markdown generator has been updated to also update all of the recipes. The next time we do a release - this will appear for all recipes.
